<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1645</width>
    <height>851</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="win_order_path">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>접수경로</string>
    </property>
   </widget>
   <widget class="QPushButton" name="win_cuckoo">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>120</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>쿠쿠</string>
    </property>
   </widget>
   <widget class="QPushButton" name="win_lg">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>LG</string>
    </property>
   </widget>
   <widget class="QPushButton" name="win_ruhens">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>루헨스</string>
    </property>
   </widget>
   <widget class="QPushButton" name="win_find">
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>90</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>조회</string>
    </property>
   </widget>
   <widget class="QPushButton" name="win_append">
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>740</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>신규 등록</string>
    </property>
   </widget>
   <widget class="QPushButton" name="win_modify">
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>140</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>수정</string>
    </property>
   </widget>
   <widget class="QPushButton" name="win_int">
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>590</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>초기화</string>
    </property>
   </widget>
   <widget class="QPushButton" name="win_gift_append">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>사은품 추가</string>
    </property>
   </widget>
   <widget class="QPushButton" name="win_model_delete">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>120</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>삭제</string>
    </property>
   </widget>
   <widget class="QPushButton" name="win_gift_delete">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>590</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>삭제</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="win_model_treeview">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>1401</width>
      <height>231</height>
     </rect>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideMiddle</enum>
    </property>
    <property name="autoExpandDelay">
     <number>100</number>
    </property>
    <attribute name="headerStretchLastSection">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string>제품사</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>주문 상태</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>설치 요청일</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>설치 완료일</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>모델명</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>렌탈료</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>색상</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>패키지</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>계약자</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>연락처</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>생년월일</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>설치주소</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>조리수 벨브</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>지불 방법</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>은행/카드사</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>계좌/카드번호</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>유효기간/예금주</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>수수료</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>폐 정수기 수거 요청</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>크로스 패키지</string>
     </property>
    </column>
   </widget>
   <widget class="QTextEdit" name="win_employee_remark">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>650</y>
      <width>391</width>
      <height>131</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>650</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>사은품 주소</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="win_gift_treeview">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>470</y>
      <width>611</width>
      <height>151</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>분류</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string notr="true">사은품</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>단가</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>사은품 발송 여부</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>사은품 비고란</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>업체명</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>타임 스탬프</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="win_time">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="win_hyundai">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>120</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>현대</string>
    </property>
   </widget>
   <widget class="QPushButton" name="win_sk">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>120</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>SK</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="win_payback_money">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>750</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>730</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>송금액</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="win_bank_name">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>750</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>730</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>은행</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>730</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>계좌번호</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="win_account_number">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>750</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>730</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>예금주</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="win_account_owner">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>750</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="win_gift_address">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>640</y>
      <width>611</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="win_call">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>  Direct</string>
    </property>
   </widget>
   <widget class="QPushButton" name="win_lg_enroll_btn">
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>480</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>전산 등록</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>최초 작성인</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>최종 수정인</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="win_last">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>30</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="win_enroll">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>30</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>전산 등록인</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="win_first">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>30</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>10</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>1400</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>상담자</string>
    </property>
   </widget>
   <widget class="QPushButton" name="win_gift_force_enroll_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>임의 등록</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="win_gift_force_enroll">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="win_chungho">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>120</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>청호</string>
    </property>
   </widget>
   <widget class="QPushButton" name="win_kyowon">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>120</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>교원</string>
    </property>
   </widget>
   <widget class="QPushButton" name="win_welrix">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>120</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>웰릭스</string>
    </property>
   </widget>
   <widget class="QPushButton" name="win_novita">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>120</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>노비타</string>
    </property>
   </widget>
   <widget class="QPushButton" name="win_samsung">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>120</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>위니아</string>
    </property>
   </widget>
   <widget class="QPushButton" name="win_install_complete_btn">
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>530</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>설치 완료</string>
    </property>
   </widget>
   <widget class="QPushButton" name="win_cherish">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>120</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>체리쉬</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="win_customer_remark">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>480</y>
      <width>391</width>
      <height>131</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>460</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>고객 비고란</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>630</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>직원 비고란</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>-40</x>
      <y>430</y>
      <width>1631</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>690</y>
      <width>771</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>1641</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>480</y>
      <width>231</width>
      <height>301</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;상태창&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="win_order_number">
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>370</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="accessibleDescription">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>350</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>주문 번호</string>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>440</y>
      <width>20</width>
      <height>371</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="win_remittance_state">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>750</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>730</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>송금 여부</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="win_remittance_state_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>750</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>730</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>송금 비고란</string>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>700</y>
      <width>20</width>
      <height>121</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="win_modify_order_state">
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>230</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>배송 상태 변경</string>
    </property>
   </widget>
   <widget class="QComboBox" name="win_order_state">
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>280</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <item>
     <property name="text">
      <string>등록 대기</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>예약</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>등록 완료</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>취소</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>설치 보류</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>설치 완료</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>재확인 요청</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="win_fees">
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>690</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="accessibleDescription">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>1510</x>
      <y>670</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>차액</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1645</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
